DeusEx/Classes/DXRFlags.uc

```diff
@@ -105,7 +105,7 @@ function LoadFlags()
     }
 
     LogFlags("LoadFlags");
-    dxr.Player.ClientMessage("Deus Ex Randomizer "$VersionString()$" seed "$seed);
+    dxr.Player.ClientMessage("Deus Ex Randomizer " $ VersionString() $ " seed: " $ seed $ ", difficulty: " $ dxr.Player.CombatDifficulty $ ", flags: " $ FlagsHash() );
```

Comment:
I figure showing these things will reduce the possibility of cheating cause they won't be able to sneakily change a flag? I guess they could always just change the code and recompile though lol

Reply:
It can't hurt.  Personally, I don't think you should worry too much about someone just changing it and recompiling.  There's pretty much nothing you can do to stop that without un-opensourcing the project (and even then...).